DesktopModules/AgapeConnect/Stories/ViewStory.ascx.vb

```diff
@@ -34,7 +37,7 @@ Namespace DotNetNuke.Modules.FullStory
             If (String.IsNullOrEmpty(story.Headline) Or (Not story.IsVisible And Not DotNetNuke.Security.Permissions.ModulePermissionController.CanEditModuleContent(Me.ModuleConfiguration))) Then
                 Response.Redirect(NavigateURL(PortalSettings.Current.ErrorPage404))
             Else
```

Comment:
There is now nothing in this else. The commented line should probably be deleted along with the Else

Reply:
Commented code removed. Else is still used.